iati/core/resources.py

```diff
@@ -261,6 +370,50 @@ def get_folder_name_for_version(version=None):
         raise ValueError("Version {} is not a valid version of the IATI Standard.".format(version))
 
 
+def get_ruleset_path(name, version=None):
+    """Determine the path of a ruleset with the given name.
+
+    Args:
+        name (str): The name of the ruleset to locate.
+        version (str): The version of the Standard to return the Ruleset for. Defaults to None. This means that paths to the latest version of the Ruleset are returned.
+
+    Returns:
+        str: The path to a file containing the specified ruleset.
+
+    Note:
+        Does not check whether the specified ruleset actually exists.
+
+    Todo:
+        Test this.
+
+    """
+    return get_path_for_version(os.path.join(PATH_RULESETS, '{0}'.format(name) + FILE_RULESET_EXTENSION), version)
+
+
+def get_schema_path(name, version=None):
+    """Determine the path of a schema with the given name.
+
+    Args:
+        name (str): The name of the schema to locate.
+        version (str): The version of the Standard to return the Schemas for. Defaults to None. This means that paths to the latest version of the Schemas are returned.
+
+    Returns:
+        str: The path to a file containing the specified schema.
+
+    Note:
+        Does not check whether the specified schema actually exists.
+
+    Warning:
+        Further exploration needs to be undertaken in how to handle multiple versions of the Standard.
+
+    Todo:
+        Handle versions of the standard other than 2.02.
```

Comment:
When merged into dev, this should work, since the underlaying `iati.core.resources.get_folder_name_for_version` is tested against versions `1.04` -> `2.02`.